src/valkey-benchmark.c

```diff
@@ -1472,6 +1481,14 @@ int parseOptions(int argc, char **argv) {
             if (lastarg) goto invalid;
             config.sslconfig.ciphersuites = strdup(argv[++i]);
 #endif
+#endif
+#ifdef USE_RDMA
+        } else if (!strcmp(argv[i], "--rdma")) {
+            if (valkeyInitiateRdma() != VALKEY_OK) {
+                fprintf(stderr, "Failed to initialize RDMA support from libvalkey\n");
+                exit(1);
+            }
+            config.ct = VALKEY_CONN_RDMA;
```

Comment:
I would suggest to place the error here that rdma option was provided for executable not supporting rdma